lib/dslash_policy.cuh

```diff
@@ -1816,14 +1836,12 @@ struct DslashFactory {
 	   QudaDslashPolicy dslash_policy = static_cast<QudaDslashPolicy>(policy_);
 
 	   // check this is a valid policy choice
-	   if ( (dslash_policy == QUDA_GPU_COMMS_DSLASH || dslash_policy == QUDA_FUSED_GPU_COMMS_DSLASH) && !comm_gdr_enabled() ) {
+	   if ( (dslash_policy == QUDA_GDR_DSLASH || dslash_policy == QUDA_FUSED_GDR_DSLASH ||
+		 dslash_policy == QUDA_GDR_RECV_DSLASH || dslash_policy == QUDA_FUSED_GDR_RECV_DSLASH)
+		&& !comm_gdr_enabled() ) {
 	     errorQuda("Cannot select a GDR policy %d unless QUDA_ENABLE_GDR is set", dslash_policy);
 	   }
 
-	   if ((dslash_policy == QUDA_ZERO_COPY_DSLASH || dslash_policy == QUDA_FUSED_ZERO_COPY_DSLASH) && comm_peer2peer_enabled_global()) {
```

Comment:
the dependence of the relation of the value written in the tunecache for the policy on the enabled Dslashes still gives me some headaches. We can leave it like that for now but when cleaning up I'd opt for something unique.

Reply:
I'll have a think about this tonight for a more absolute and less relative method.

Reply:
I have a first implementation in #666